shared/src/business/useCases/health/getHealthCheckInteractor.js

```diff
@@ -0,0 +1,211 @@
+const regionEast = 'us-east-1';
+const regionWest = 'us-west-1';
+
+const handleAxiosTimeout = axios => {
+  let source = axios.CancelToken.source();
+  setTimeout(() => {
+    source.cancel();
+  }, 1000);
+  return source;
+};
+
+const getElasticSearchStatus = async ({ applicationContext }) => {
+  try {
+    await applicationContext.getPersistenceGateway().getFirstSingleCaseRecord({
+      applicationContext,
+    });
+  } catch (e) {
+    console.log('Elasticsearch health check failed. ', e);
+    return false;
+  }
+
+  return true;
+};
+
+const getDynamoStatus = async ({ applicationContext }) => {
+  try {
+    const status = await applicationContext
+      .getPersistenceGateway()
+      .getTableStatus({ applicationContext });
+    return status === 'ACTIVE';
+  } catch (e) {
+    console.log('Dynamo health check failed. ', e);
+    return false;
+  }
+};
+
+const getDeployDynamoStatus = async ({ applicationContext }) => {
+  try {
+    const status = await applicationContext
+      .getPersistenceGateway()
+      .getDeployTableStatus({ applicationContext });
+    return status === 'ACTIVE';
+  } catch (e) {
+    console.log('Dynamo deploy health check failed. ', e);
+    return false;
+  }
+};
+
+const getDynamsoftStatus = async ({ applicationContext }) => {
+  const axios = applicationContext.getHttpClient();
+
+  const source = handleAxiosTimeout(axios);
+
+  try {
+    // Currently, dynamsoft is only deployed on the court's staging environment as it shares a license with
```

Comment:
Is there an issue in the backlog/Trello to address this?

Reply:
@mmarcotte is going to look further into whether there's a need for another Dynamsoft account - marking this comment as resolved.